src/fluree/db/query/exec/where.cljc

```diff
@@ -12,13 +12,21 @@
 
 #?(:clj (set! *warn-on-reflection* true))
 
+(defn reference?
+  [mch]
```

Comment:
I think `mch` is an abbreviation for "match", but it took me a few seconds to come up with that. I don't think we're saving too much space by abbreviating here and just using the full name would make it easier to read in the future.